metrikker/src/main/java/no/nav/familie/metrikker/TellAPIEndepunkterIBrukInitialiserer.kt

```diff
@@ -0,0 +1,71 @@
+package no.nav.familie.metrikker
+
+import io.micrometer.core.instrument.Counter
+import io.micrometer.core.instrument.Metrics
+import jakarta.annotation.PostConstruct
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty
+import org.springframework.context.ApplicationContext
+import org.springframework.stereotype.Component
+import org.springframework.web.method.HandlerMethod
+import org.springframework.web.servlet.mvc.method.RequestMappingInfo
+import org.springframework.web.servlet.mvc.method.annotation.RequestMappingHandlerMapping
+
+@Component
+@ConditionalOnProperty("familie.tellAPIEndepunkterIBruk")
+class TellAPIEndepunkterIBrukInitialiserer(
+    @Value("\${NAIS_APP_NAME}") private val applicationName: String,
+    private val applicationContext: ApplicationContext,
+) {
+    init {
+        metrikker.clear()
+    }
+
+    @PostConstruct
+    fun populerMapMedCountersForRestEndepunkt() {
+        val requestMappingHandlerMapping: RequestMappingHandlerMapping =
+            applicationContext
+                .getBean("requestMappingHandlerMapping", RequestMappingHandlerMapping::class.java)
+        val requestMappings: Map<RequestMappingInfo, HandlerMethod> = requestMappingHandlerMapping.handlerMethods
+
+        requestMappings.forEach { (info, handler) ->
+            info.patternValues.forEach { path ->
+                if (path.startsWith("/api")) {
```

Comment:
Gjelder dette for alle repoene i familie?

Reply:
Ja, men kan muligens trekke det ut i config